src/app/main/header/header.component.scss

```diff
@@ -41,6 +43,28 @@
   margin: 0 16px;
 }
 
+.other-search-link {
+    display: grid;
```

Comment:
Why do you use grid here?

Reply:
because I prefer this solution to get a blank space > button advanced > button expert, instead of using a class "fill-in-remaining-space" (I tried it though but I could not make it look good as I wanted)
and one day if we want to move the project filter menu here, the grid will be helpful to position it well

Reply:
Thanks for explanation